debian/usr.bin.snap-confine

```diff
@@ -112,6 +114,15 @@
     mount options=(rw bind) /snap/ubuntu-core/*/libx32/ -> /libx32/,
     mount options=(rw bind) /snap/ubuntu-core/*/lib64/ -> /lib64/,
     mount options=(rw bind) /snap/ubuntu-core/*/usr/ -> /usr/,
+    mount options=(rw bind) /snap/ubuntu-core/*/etc/alternatives/ -> /etc/alternatives/,
+
+    # This is a somewhat of a blank check for mount profiles.  Mount profiles
+    # are written by the trusted snapd to a root-writable location and
+    # processed by snap-confine. Regardless of configuration options and
+    # runtime factors they are processed after pivot_root (on classic systems)
+    # so snap.rootfs is not a factor.
+    mount options=(rw bind),
+    mount options=(ro bind),
```

Comment:
FYI:
```
20:36 < jdstrand> zyga (cc tyhicks): the next step is to give examples of mount profiles that snapd 
                  currently implements
20:36 < zyga> jdstrand: only the content interface uses them
20:36 < zyga> jdstrand: and the read/write paths are decided by plug/slot attributes
20:36 < jdstrand> zyga (cc tyhicks): and then understand precisely what mount rules are needed
20:36 < zyga> jdstrand: any value is allowed right now
20:37 < zyga> jdstrand: that fact implies that any mount rules might be allowed
20:37 < jdstrand> zyga: yes, but the source mount point is inside a snap, not /**
20:37 < jdstrand> zyga: and the target mount point is inside a snap, not /**
20:37 < zyga> jdstrand: one useful use case is /var/lib/snapd/hostfs/usr/share/fonts 
20:37 < zyga> jdstrand: as for the target directory, let me check
20:38 < jdstrand> zyga: but the fonts directory is an implicit slot, not the content interface
20:38 < tyhicks> zyga: sorry to distract from the discussion but does the snap itself define the 
                 plug/slot attributes?
20:38 < zyga> jdstrand: it would be a slot on the core snap (it doesn't exist yet)
20:38 < zyga> tyhicks: yes
20:38 < zyga> small clarification, currently both source and destination are rooted at $SNAP
20:38 < jdstrand> zyga: right, so we add a rules for /usr/share/fonts like we do for /etc/alternatives
20:39 < zyga> so I guess we can constrain those profiles to /snap/*/**
20:39 < jdstrand> zyga: and then a different glob rule for content
20:39 < jdstrand> yes! :)
20:39 < zyga> I missed those two lines, I just read the content interface
```